examples/models/qwen3/qwen3_32b_decode_fused_qkrope.py

```
@@ -0,0 +1,781 @@
+  2. Fused Q projection + Q RoPE + q pad
+  3. K RoPE + cache write during K projection tiling
+  4. V cache write during V projection tiling
+
+Scope 2:
+  1. QK matmul
+  2. Softmax
+  3. SV matmul
+  4. Online-softmax accumulation + final normalisation
+
+Scope 3:
+  1. Output projection: attn_out × wo
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)